src/main/sensors/initialisation.c

```diff
@@ -734,7 +734,7 @@ static void detectMag(magSensor_e magHardwareToUse)
 
     if (magHardware == MAG_NONE && magHardwareToUse != MAG_DEFAULT && magHardwareToUse != MAG_NONE) {
         // Nothing was found and we have a forced sensor that isn't present.
-        magHardwareToUse = MAG_DEFAULT;
+        magHardwareToUse = MAG_NONE;
```

Comment:
Sorry, could not hold myself...

![goto](https://cloud.githubusercontent.com/assets/966811/16795596/79941974-48de-11e6-9739-623888f44240.png)